frontend/apps/web/src/pages/tests.tsx

```diff
@@ -97,10 +96,26 @@ const AllTests = () => {
 
   const prepareAllTestsStatus = () => {
     return Object.entries(
-      allTests.reduce((d, test) => {
-        d[test.status] = (d[test.status] || 0) + 1
-        return d
-      }, {}),
+      allTests.reduce(
+        (d, test) => {
+          if (test.status === 'pass') {
+            d.Passing += 1
+          } else if (test.status === 'fail') {
+            d.Failing += 1
+          } else if (test.status === 'missing') {
+            d.Missing += 1
+          } else {
+            d.Unknown += 1
```

Comment:
We don't have Unknown test status, we can remove this.

Reply:
Done